client/src/App.js

```diff
@@ -28,7 +28,7 @@ export default class App extends Component {
   render () {
     return (
       <div>
-        <div style={{display: 'flex', flexDirection: 'row', height: '70em'}}>
+        <div style={{display: 'flex', flexDirection: 'row', height: '100vh'}}>
```

Comment:
Makes this `div` take up 100% of the screen estate, all the time. See `vh` at https://developer.mozilla.org/fr/docs/Web/CSS/length